operations/research-and-development/product/product-planning/release-plan.md

```diff
@@ -1,11 +1,30 @@
+## Release Plan vs. Roadmap
 
-We follow a time-based release philosophy.  This means we release a new version of our Mattermost product every month on the 16th. On even numbered months (February, April, June, August, October, and December). we release new features in our product. On odd numbered months (January, March, May, July, September, and November), we do not release features, but improve the quality and stability of features previously released by only releasing bug fixes. We recommend customers who value stability, to upgrade following our quality or odd-numbered releases. Each month’s release includes a [release blog](https://mattermost.com/blog/category/releases/), which highlights features or improvements and updates to our [changelog](https://docs.mattermost.com/administration/changelog.html), where all changes can be viewed for the release. 
+A release plan is different from a roadmap, in that a roadmap typically lays out the direction and prioritization the product is headed and includes larger objectives and larger features. 
 
-Our quality releases for January and July, are offered as an [Extended Support Releases (ESR)](https://docs.mattermost.com/administration/extended-support-release.html). This type of release allows customers to receive backports for security fixes and bug fixes without having to upgrade to a version with new functionality that would require significant testing and certification before rolling to large volume of users. We also do dot releases for major bugs and security fixes and back port these to the previous three monthly releases. For more information on our release types and processes please see this [documentation](https://docs.mattermost.com/process/release-faq.html).
+The release plan may include major bugs, smaller features and even tasks required to complete a feature set. The release plan document typically includes all the R&D teams’ proposed work so that we can get a holistic view of what features, bugs or improvements are targeted for the release. Each team may also have their own version of a release plan that they use for tracking features and resource planning. You can find our [release plan here](https://docs.google.com/spreadsheets/d/1K7b7hIfYHz-jbzonJlN1vosiWZRbU1fb5aZ_2xO7tE0/edit#gid=420927153).
```

Comment:
```suggestion
The release plan may include major bugs, smaller features and even tasks required to complete a feature set. The release plan document typically includes all the R&D teams’ proposed work so that we can get a holistic view of what features, bugs or improvements are targeted for the release. Each team may also have their own version of a release plan that they use for tracking features and resource planning. You can find our [release plan here](https://mattermost.productboard.com/feature-board/1097526-release-tracking-internal).
```